src/controller/python/chip/server/ServerInit.cpp

```diff
@@ -99,23 +100,40 @@ void CleanShutdown()
 using PostAttributeChangeCallback = void (*)(EndpointId endpoint, ClusterId clusterId, AttributeId attributeId, uint8_t type,
                                              uint16_t size, uint8_t * value);
 
+using PreAttributeChangeCallback = chip::Protocols::InteractionModel::Status (*)(EndpointId endpoint, ClusterId clusterId,
```

Comment:
how do we handle different status codes (IM::Status vs EmberAfStatus) returned from Python? Should we have constants for them or are such constants already defined?

Could we have that documented somewhere on what common statuses mean to be returned from the callback?